tests/Comparator/Int64Comparator.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace MongoDB\Tests\Comparator;
```

Comment:
This class could be in the `src` directory, with namespace `MongoDB\Test\Comparator`. The need to compare Int64 also exist for the developers using this lib.

Reply:
Understood, exposing such PHPUnit extension requires that we think about the supported PHPUnit versions BC promise. If user feels the need for this comparator, please raise an issue or add a comment to the new task PHPLIB-1165.